pkg/image/installer_image_test.go

```diff
@@ -593,3 +593,91 @@ func TestTarInstallerModules(t *testing.T) {
 		}
 	}
 }
+
+func findAnacondaLocale(t *testing.T, mf manifest.OSBuildManifest) string {
+	pipeline := findPipelineFromOsbuildManifest(t, mf, "anaconda-tree")
+	assert.NotNil(t, pipeline)
+	stage := findStageFromOsbuildPipeline(t, pipeline, "org.osbuild.locale")
+	localeStageOptions := stage["options"].(map[string]interface{})
+	language := localeStageOptions["language"].(string)
+	return language
+}
+
+func TestInstallerLocales(t *testing.T) {
+	assert := assert.New(t)
+
+	for _, lang := range []string{"C.UTF-8", "invalid", "el_CY.UTF-8", "en_GB.UTF-8", ""} {
+		expected := lang
+		if expected == "" {
+			expected = "C.UTF-8" // default
+		}
```

Comment:
Nitpick: you could eliminate the special case here as well with having the expected and configured values captured as `map[string]string`

Reply:
You're right, I was being sloppy and lazy.  Thanks :)